gitlab/v4/objects.py

```diff
@@ -1250,6 +1250,17 @@ def enable(self, key_id, **kwargs):
         self.gitlab.http_post(path, **kwargs)
 
 
+class ProjectBadge(SaveMixin, ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class ProjectBadgeManager(CRUDMixin, RESTManager):
+    _path = '/projects/%(project_id)s/badges'
+    _obj_cls = ProjectBadge
+    _from_parent_attrs = {'project_id': 'id'}
+    _create_attrs = (('link_url', 'image_url'), tuple())
```

Comment:
It is also possible to update the badge objects, so could you define the `_update_attrs` variable as well?